lib/services/database.go

```diff
@@ -809,6 +809,60 @@ func NewDatabasesFromRDSClusterCustomEndpoints(cluster *rds.DBCluster) (types.Da
 	return databases, trace.NewAggregate(errors...)
 }
 
+// NewDatabasesFromRDSCluster creates all database resources from an RDS Aurora
+// cluster.
+func NewDatabasesFromRDSCluster(cluster *rds.DBCluster) (types.Databases, error) {
```

Comment:
Moved logic from db fetcher to `services` so it's easier for another package to do the same.